CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,73 @@
+  project once.
+- **respect** your abilities, interests, and personal differences.  Our
+  different talents documenting, testing, coding, reviewing, reporting, and/or
+  evagenlizing are all valuable to the project.
+- **make concise and actionable points** to respect your time as a volunteer.
+  We respect each other's time by keeping the signal-to-noise ratio high,
+  staying on-topic, and being a good house-guest: every visit we strive to
+  leave the place in better shape than when we arrived.
+- **are kind** and **assume positive intent**, even when we disagree while
+  working towards consensus and achieving the project's technical goals.
+- **educate and illuminate** others with something we know more about. We were
+  all beginners once, and can teach and learn through feedback and sharing.
+- **are open to feedback** from others, be it technical points,  interpersonal
+  tips, or language/wording suggestions.
+- **use words carefully**, because we may not understand irony or negative
+  sarcasm. We communicate plainly, and we justify our positions objectively and
+  explicitly.
+- **leave with class**, without drama or bitterness, when resigning for any
+  reason. We champion your rights under the terms of the GPL to fork the code
+  and create a new project, just as we stand on the shoulders of those who gave
+  us the right to create this project.
+
+## Reporting Issues
+
+Please bring up issues by contacting
+[dreamer](mailto:patryk.obara+coc@gmail.com) or
+[krcroft](mailto:k.r.croft+coc@gmail.com) who will handle your report with
+discretion. Your report should include:
+
+- Your preferred contact information. We cannot process anonymous reports.
+- Names (real or usernames) of those involved in the incident.
+- Your description of what occurred, and if the incident is ongoing. Please
+  provide links to or transcripts of the publicly available records (e.g. a
+  mailing list archive, GitHub issue, or a public IRC logger), so that we can
+  review it.
+- Any additional information that may be helpful to achieve resolution.
+
+All reports are confidential.
+
+### Scope
+
+This goal of this document is to preserve the well being of the project and its
+community, and we will will restrict or remove participation from those who
+disrupt the project and its participants.
+
+This code does not replace the terms of service or acceptable use policies that
+are provided by the websites used to support this community. Nor does this code
+apply to communications or actions that take place outside of the context of
+this community. Many participants in this project are also subject to codes of
+conduct based on their employment. This code is a social-contract that informs
+participants of our social expectations. It is not a terms of service or legal
+contract.
+
+## License and Acknowledgement
+
+This text is shared under the [CC-BY-4.0
+license](https://creativecommons.org/licenses/by/4.0/).
```

Comment:
CC-BY-4.0 requires attribution (credit) to the author. How about using a license closer to public domain like CC0?

Reply:
Works for me!